libraries/AP_TECS/AP_TECS.cpp

```diff
@@ -1220,15 +1220,17 @@ void AP_TECS::update_pitch_throttle(int32_t hgt_dem_cm,
     // @Field: PErr: difference between estimated potential energy and desired potential energy
     // @Field: EDelta: current error in speed/balance weighting
     // @Field: LF: aerodynamic load factor
-    AP::logger().Write("TEC2", "TimeUS,pmax,pmin,KErr,PErr,EDelta,LF",
-                       "s------",
-                       "F------",
-                       "Qffffff",
+    // @Field: EAS2TAS: equivelent to true airspeed ratio
+    AP::logger().Write("TEC2", "TimeUS,pmax,pmin,KErr,PErr,EDelta,LF,EAS2TAS",
```

Comment:
this is useful in non-TECS modes, such as FBWA, so CTUN seems like a good place

Reply:
moved to CTUN